dev/reference/sql/character-set.md

```diff
@@ -207,4 +207,10 @@ You can use the following statement to specify a particular collation that is re
     SET collation_connection = @@collation_database;
     ```
 
+## Validity check of characters
+
+For the specified `utf8` or `utf8mb4` character set, TiDB only supports the valid `utf8` character, and reports the `incorrect utf8 value` error when the character is invalid. This validity check of characters in TiDB is compatible with MySQL 8.0 but incompatible with MySQL 5.7 or earlier versions.
+
+To mute this error reporting, use `set @@tidb_skip_utf8_check=1;` to skip the character check.
```

Comment:
```suggestion
To disable this error reporting, use `set @@tidb_skip_utf8_check=1;` to skip the character check.
```
"Disable/turn off the error reporting" is more commonly used if you search it in Google.